saleor/plugins/webhook/utils.py

```diff
@@ -138,3 +149,71 @@ def parse_list_shipping_methods_response(
             )
         )
     return shipping_methods
+
+
+@contextmanager
+def catch_duration_time():
+    start = datetime.now()
+    yield lambda: datetime.now() - start
```

Comment:
```suggestion
from time import time
@contextmanager
def catch_duration_time():
    start = time()
    yield lambda: time() - start
```
Maybe we should stick to `time` module instead of `datetime` to prevent problems regarding counting time delta during a change to daylight saving time?